### PR TITLE
Add skywalking agent deployment docs

### PR DESCRIPTION
**Pull request by hailin0**
Add skywalking agent deployment docs

[reference PR](https://github.com/apache/incubator-dolphinscheduler/pull/4852)

### CODE REVIEW

**Comment by hailin0** [comment]
@dailidong @CalvinKirs

**Comment by dailidong** [comment]
good job, I will merge this PR when 1.3.6 released

File: docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+添加以下配置到 `${workDir}/conf/config/install_config.conf`.
+
+```properties
+
+# skywalking config
+# note: enable skywalking tracking plugin
+enableSkywalking="true"
+# note: configure skywalking backend service address
+skywalkingServers="your.skywalking-oap-server.com:11800"
```
Comment (lgcareer) [inline docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md:37]: Good job!Hi,what is different form skywalkingServers and skywalkingLogReporterHost?If they are the same host ,is it ok?

**Comment by wu-sheng** [comment]
There is no preview here? ...

File: docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+
+# 使用
+
+### 导入图表
+
+#### 导入图表模板到 Skywalking server
+
+导入 `${dolphinscheduler.home}/skywalking-agent/dashboard/dolphinscheduler.yml` 文件到 `${skywalking-oap-server.home}/config/ui-initialized-templates/` 目录下，并重启 Skywalking oap-server.
```
Comment (wu-sheng) [inline docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md:51]: 如果已经在使用SkyWalking，应该还需要清空浏览器缓存？

File: docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md (hunk 1 above)
Comment (hailin0) [reply docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md:37]:
```text
skywalkingServers  ==>  collector.backend_service
skywalkingLogReporterHost  ==>  plugin.toolkit.log.grpc.reporter.server_host
skywalkingLogReporterPort  ==>  plugin.toolkit.log.grpc.reporter.server_port
```

The configuration items are from [skywalking docs](https://github.com/apache/skywalking/tree/master/docs/en/setup/service-agent/java-agent).

File: docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md (hunk 2 above)
Comment (hailin0) [reply docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md:51]: 好的，补充了

**Comment by hailin0** [comment]
> There is no preview here? ...

image
image

**Review by chengshiwen — APPROVED** [review]
+1

**Pull request merged**